rest-api/dao/participant_dao.py

```diff
@@ -151,25 +151,21 @@ def _do_update(self, session, obj, existing_obj):
     super(ParticipantDao, self)._do_update(session, obj, existing_obj)
 
 
-  def get_pairing_level(self, obj, existing_obj):
+  def get_pairing_level(self, obj):
     organization_id = obj.organizationId
     site_id = obj.siteId
     awardee_id = obj.hpoId
-    old_site_id = existing_obj.siteId
-    old_organization_id = existing_obj.organizationId
     # TODO: DO WE WANT TO PREVENT PAIRING IF EXISTING SITE HAS PM/BIO.
 
     if site_id != u'UNSET' and site_id is not None:
-      if old_site_id != site_id:
-        site = self.site_dao.get(site_id)
-        organization_id = site.organizationId
-        awardee_id = site.hpoId
-        return site_id, organization_id, awardee_id
-    if organization_id != u'UNSET' and organization_id is not None:
-      if organization_id != old_organization_id:
-        organization = self.organization_dao.get(organization_id)
-        awardee_id = organization.hpoId
-        return None, organization_id, awardee_id
+      site = self.site_dao.get(site_id)
+      organization_id = site.organizationId
+      awardee_id = site.hpoId
+      return site_id, organization_id, awardee_id
+    elif organization_id != u'UNSET' and organization_id is not None:
+      organization = self.organization_dao.get(organization_id)
```

Comment:
Similarly, we should throw BadRequest here if organization is None.

Reply:
done